dynamic_rest/renderers.py

```diff
@@ -91,6 +91,9 @@ def process(result):
             c for c in context['columns'] if c not in self.COLUMN_BLACKLIST
         ]
         context['details'] = context['columns']
+        context['error_form'] = context['error_form']
+        if context['error_form']:
```

Comment:
it shouldn't. 

it shouldn't! [see here](
https://github.com/AltSchool/dynamic-rest/pull/160/files#diff-c6db66dec7f23eb26c7417373b9160b0R244)

![screen shot 2017-03-29 at 4 24 33 pm](https://cloud.githubusercontent.com/assets/4039710/24480983/68a4d5b0-149c-11e7-85e0-d6dfb766dca4.png)